src/page-modules/contact/components/input/input.module.css

```diff
@@ -0,0 +1,41 @@
+@value typo-body__secondary from "@atb/theme/typography.module.css";
+
+.container {
+  composes: typo-body__secondary;
+
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  color: var(--static-background-background_0-text);
+  background-color: var(--static-background-background_0-background);
+  gap: var(--spacings-small);
+  margin-bottom: var(--spacings-medium);
+}
+
+.radio_or_checkbox {
```

Comment:
I don't think that the class name .radio_or_checkbox is descriptive enough. Instead, the class name should describe the purpose or function of the class, rather than just indicating the types of elements it might be used for